ray_on_golem/ctl/ctl.py

```diff
@@ -0,0 +1,149 @@
+import subprocess
+import time
+from datetime import datetime
+from pathlib import Path
+from typing import TYPE_CHECKING, Optional
+
+from ray_on_golem.log import ZippingRotatingFileHandler
+from ray_on_golem.server.models import ShutdownState
+from ray_on_golem.server.settings import (
+    LOGGING_BACKUP_COUNT,
+    RAY_ON_GOLEM_CHECK_DEADLINE,
+    RAY_ON_GOLEM_PATH,
+    RAY_ON_GOLEM_SHUTDOWN_DEADLINE,
+    RAY_ON_GOLEM_START_DEADLINE,
+    get_log_path,
+)
+from ray_on_golem.utils import get_last_lines_from_file
+
+if TYPE_CHECKING:
+    from ray_on_golem.client import RayOnGolemClient
+    from ray_on_golem.ctl.log import RayOnGolemCtlLogger
+
+
+class RayOnGolemCtl:
+    def __init__(self, client: "RayOnGolemClient", output_logger: "RayOnGolemCtlLogger"):
+        self.client = client
+        self.output_logger = output_logger
```

Comment:
Is there any reason to expose those fields as a public fields?

Reply:
maybe indeed, the `output_logger` doesn't have to be exposed, although, on the other hand, it makes the usage a bit simpler

Reply:
updated